smooks-core/src/main/java/org/milyn/payload/JavaResult.java

```diff
@@ -42,7 +42,7 @@
      * Public default constructor.
      */
     public JavaResult() {
-        this(false);
+        this(true);
```

Comment:
Okay ... I'll take your word for it (and blame you if others run into issues because of it 😉  )